pages/workspaces/[workspaceId]/activity.tsx

```diff
@@ -22,10 +22,13 @@ import WorkspaceBanner from "components/Workspaces/WorkspaceBanner";
 import { SubTabsList } from "components/TabList/tab-list";
 import { StarSearchEmbed } from "components/StarSearch/StarSearchEmbed";
 import { useMediaQuery } from "lib/hooks/useMediaQuery";
-import { FeatureFlag, getAllFeatureFlags } from "lib/utils/server/feature-flags";
 import { WORKSPACE_STARSEARCH_SUGGESTIONS } from "lib/utils/star-search";
+import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
+import { useWorkspaceMembers } from "lib/hooks/api/useWorkspaceMembers";
 
-const InsightUpgradeModal = dynamic(() => import("components/Workspaces/InsightUpgradeModal"));
+const InsightUpgradeModal = dynamic(() => import("components/Workspaces/InsightUpgradeModal"), {
+  ssr: false,
```

Comment:
Don't need to server-side render this. as it's a modal that might never get used, so load it client-side.